com.unity.shadergraph/Documentation~/surface-options.md

```diff
@@ -0,0 +1,38 @@
+# Modify surface options without changing your graph
+
+## Description
+
+Enable **Allow Material Override** to modify a specific set of properties for Universal Render Pipeline Lit and Unlit Shader Graphs and for Built-In Render Pipeline Shader Graphs in the Material Inspector.
+
+This functionality makes it possible for you to override the following Universal Render Pipeline Lit shader properties:
+
+* Workflow Mode
+* Surface Type
+* Render Face
+* Depth Write
+* Depth Test
+* Alpha Clipping
+* Cast Shadows
+* Receive Shadows
+
+These are the Universal Render Pipeline Unlit shader properties you can override:
```

Comment:
Which properties are exposed for the Built-in Render Pipeline? We should list them too, in the same way as we listed for URP Lit and Unlit.

Reply:
I'm not sure which ones and I am not sure how to pull a version that includes this functionality so I can look. I was trying to look in the code PR but that was a challenge too. @joshua-davis Can you let me know?

But in the meantime I put the table together . . .

Reply:
Same as URP except no Workflow Mode, Cast Shadows, or Receive Shadows options.
So Surface Type, Render Face, Depth Write, Depth Test, Alpha Clipping, Blending Mode (if transparent Surface Type).